src/plugin.js

```diff
@@ -228,70 +228,22 @@ const onPlayerReady = (player, emeError) => {
 
   setupSessions(player);
 
-  if (window.MediaKeys) {
+  const playerOptions = getOptions(player);
+  const isLegacyFairplay = playerOptions.keySystem && playerOptions.keySystem[LEGACY_FAIRPLAY_KEY_SYSTEM];
+
+  if (window.MediaKeys && !isLegacyFairplay) {
```

Comment:
I could be off here, and I guess this is just for my own understanding.. but should we check if this is the newer version of Fairplay, and not just NOT the legacy version?

Reply:
@wseymour15, @dzianis-dashkevich 's comment is spot on and exactly the reason for the check. Safari supports both versions, legacy fairplay (`com.apple.fps.1_0`) uses a different flow through contrib-eme than newer EME compliant fairplay (`com.apple.fps`), which shares the same flow as other keysystems that comply with the EME spec.